pyodc/codec.py

```diff
@@ -151,6 +151,66 @@ def from_dataframe(cls, column_name: str, data: pd.Series, data_type: DataType,
     def decode(self, stream):
         return struct.pack("<d", self.min).split(b"\x00", 1)[0].decode("utf-8")
 
+class VariableLengthConstantString(Constant):
+    "This deals with constant string columns where the value is longer than 64 bits"
+    def __init__(
+        self,
+        column_name: str,
+        constant_string: str,
+    ):
+        self.column_name = column_name
+        self.has_missing = False
+        self.constant_string = constant_string
+        self._data_size = len(constant_string.encode("utf-8"))
+
+        self.type = DataType.STRING
+        self.missing_value = MISSING_STRING
+        self.typed_missing_value = ""
+
+        assert self.name is not None
+        assert self._data_size > 8 
+
+
+    @classmethod
+    def from_dataframe(cls, column_name: str, data: pd.Series, data_type: DataType, bitfields):
+        assert data.nunique() == 1 and not data.hasnans
+        assert data_type == DataType.STRING
+        assert not bitfields
+        return cls(column_name, constant_string = next(iter(data)))
+
+    @property
+    def data_size(self):
+        return self._data_size
+
+    @property
+    def numChanges(self):
+        return 0
+
+    def encode_header(self, stream):
+        """
+        Encode the column header including both the 
+        universal (name, dtype, codec) part and the
+        addtional codec specific data.
+        """
+        stream.encodeString(self.column_name) # Column name
```

Comment:
These are already encoded in the base codec type. We should be using the base encode_header function here. i.e. super().encode_header(). Even if that does cause us to encode unnecessary/unusable other values (yuk)

Same in the decode pathway.